src/Rector/Contrib/Symfony/Controller/RedirectToRouteRector.php

```diff
@@ -91,15 +91,15 @@ private function resolveArguments(MethodCall $node): array
         $arguments = [];
         $arguments[] = $generateUrlNode->args[0];
 
-        if ($generateUrlNode->args[1]) {
+        if (isset($generateUrlNode->args[1])) {
             $arguments[] = $generateUrlNode->args[1];
         }
 
-        if (! $generateUrlNode->args[1] && $node->args[1]) {
+        if (! isset($generateUrlNode->args[1]) && isset($node->args[1])) {
             $arguments[] = [];
         }
 
-        if ($node->args[1]) {
+        if (isset($node->args[1])) {
             $arguments[] = $node->args[1];
         }
```

Comment:
Could you add tests that prove this would fail without issets? So we know it has meaning to add and is also covered.

Reply:
Is this added somewhere?